packages/core/chrome/core-chrome-browser-internal/src/ui/project/header.tsx

```diff
@@ -179,9 +167,6 @@ export const ProjectHeader = ({
   docLinks,
   ...observables
 }: Props) => {
-  const [navId] = useState(htmlIdGenerator()());
-  const [isOpen, setIsOpen] = useLocalStorage(LOCAL_STORAGE_IS_OPEN_KEY, true);
```

Comment:
FYI - you can still store/fetch this information in localStorage using `<EuiCollapsibleNavBeta initialIsCollapsed={someLocalStorageData}>`. Just want to make sure we're not losing any UX in the new nav.

... Although on reflection, I never added a `onToggleCollapse` callback to the new component 🤦 Let me add that real quick...

Reply:
Thanks so much @cee-chen! I was able to use your latest code in 9ddf71b4e88

Reply:
Perfection!!!